components/table/stories/table.stories.js

```diff
@@ -24,11 +22,15 @@ export default {
 			control: "select",
 		},
 		isQuiet,
-		isEmphasized,
+		isEmphasized: {
+			...isEmphasized,
+			description: "Emphasized styling on the table affects colors of selected rows and any checkboxes."
+		},
```

Comment:
TODO: is this control necessary for all stories? Is it ok if we disable the control per story? Now that colors are in place, revisit some of the questions Rise posed in #3799